packages/webapp/src/pages/Environment/Settings/ConnectUISettings/index.tsx

```diff
@@ -142,154 +189,57 @@ export const ConnectUISettings = () => {
-                                                            value={field.state.value}
-                                                            onChange={(e) => field.handleChange(e.target.value)}
-                                                            onBlur={field.handleBlur}
-                                                            disabled={!environment.plan?.can_customize_connect_ui_theme}
-                                                        />
-                                                        {!field.state.meta.isValid && (
-                                                            <em role="alert" className="text-sm text-red-500">
-                                                                {field.state.meta.errors.join(', ')}
-                                                            </em>
-                                                        )}
-                                                    </div>
-                                                </TooltipTrigger>
-                                                {!environment.plan?.can_customize_connect_ui_theme && (
-                                                    <TooltipContent side="bottom" className="text-grayscale-300">
-                                                        Customizing the theme is only available for Growth plans.{' '}
-                                                        <LinkWithIcon to={`/${env}/team/billing`}>Upgrade your plan</LinkWithIcon>
-                                                    </TooltipContent>
-                                                )}
-                                            </Tooltip>
-                                        )}
-                                    </form.Field>
-                                ))}
-                            </div>
-                        </div>
-                    </div>
+                                {!canCustomizeTheme && <ThemeColorPickers disabled={true} form={form} />}
+                                {!canDisableWatermark && <WatermarkToggle disabled={true} form={form} />}
 
-                    {/** Save Button */}
-                    <form.Subscribe selector={(state) => [state.canSubmit, state.isDirty, state.isSubmitting]}>
-                        {([canSubmit, isDirty]) => (
-                            <Button
-                                type="submit"
-                                variant="primary"
-                                size="md"
-                                className="self-start"
-                                disabled={!canSubmit || !isDirty}
-                                isLoading={isUpdatingConnectUISettings}
-                            >
-                                Save Connect UI settings
-                            </Button>
+                                <ButtonLink to={`/${env}/team/billing#plans`} variant="secondary" className="bg-bg-muted" target="_blank">
```

Comment:
Was this not a regular secondary button?
```suggestion
                                <ButtonLink to={`/${env}/team/billing#plans`} variant="secondary" target="_blank">
```

Reply:
No, when it's secondary it looks too much like the background.

My version:
<img width="389" height="141" alt="image" src="https://github.com/user-attachments/assets/f406f4c1-df8f-4f7e-a2ae-9bd0bb8dbda7" />

Plain secondary:
<img width="390" height="130" alt="image" src="https://github.com/user-attachments/assets/e07e09ee-eba6-4582-ac01-03c7e241498c" />